From f9d58a058023f5a0ff74dd62b3e5cc0b8329283f Mon Sep 17 00:00:00 2001
From: Mark Harfouche <mark.harfouche@gmail.com>
Date: Tue, 22 Apr 2025 15:57:34 -0400
Subject: [PATCH] [PATCH] Get package version and description from conda build

---
 utils/update_build_version.py | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/utils/update_build_version.py b/utils/update_build_version.py
index 9115cab1..2aa3d957 100755
--- a/utils/update_build_version.py
+++ b/utils/update_build_version.py
@@ -145,13 +145,9 @@ def main():
     changes_file_path = os.path.realpath(sys.argv[1])
     output_file_path = sys.argv[2]
 
-    success, version = deduce_software_version(changes_file_path)
-    if not success:
-      logging.error("Could not deduce latest release version from {}.".format(changes_file_path))
-      sys.exit(1)
+    version = os.environ.get('PKG_VERSION', 'unknown')
+    description = os.environ.get('PKG_VERSION', 'unknown') + ", " + os.environ.get('PKG_BUILDNUM', 'unkown')
 
-    repo_path = os.path.dirname(changes_file_path)
-    description = os.getenv("FORCED_BUILD_VERSION_DESCRIPTION", describe(repo_path))
     content = OUTPUT_FORMAT.format(version_tag=version, description=description)
 
     # Escape file content.
-- 
2.43.0

